lib/metanorma/standoc/processor.rb

```diff
@@ -8,8 +8,6 @@ def initialize
         @short = :standoc
         @input_format = :asciidoc
         @asciidoctor_backend = :standoc
-
```

Comment:
Only when a `[latexmath]` block is encountered we need LaTeXML. @opoudjis any tips here?